pybandits/transfer.py

```
@@ -434,16 +500,29 @@ def _expand_with_template_weights(
+    if n_new_features > 0:
+        logger.info(
+            f"Expanding current CMAB from {current_dim} to {template_dim} features "
+            f"using template's weights for {n_new_features} new feature(s)"
+        )
+
+    # Validate hidden layer architecture changes for overlapping actions
+    for action_id in set(current_cmab.actions.keys()) & set(template_cmab.actions.keys()):
+        src_model = current_cmab.actions[action_id]
+        tgt_model = template_cmab.actions[action_id]
+        if isinstance(src_model, BaseModelMO):
+            for obj_idx, (src_obj, tgt_obj) in enumerate(zip(src_model.models, tgt_model.models)):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)